e2e_batch/constants.py

```diff
@@ -12,77 +11,87 @@
-        "VACCINE_MANUFACTURER": "Sanofi Pasteur",
-        "BATCH_NUMBER": "BN92478105653",
-        "EXPIRY_DATE": "20240915",
-        "SITE_OF_VACCINATION_CODE": "368209003",
-        "SITE_OF_VACCINATION_TERM": "Right arm",
-        "ROUTE_OF_VACCINATION_CODE": "1210999013",
-        "ROUTE_OF_VACCINATION_TERM": "Intradermal use",
-        "DOSE_AMOUNT": dose_amount,
-        "DOSE_UNIT_CODE": "2622896019",
-        "DOSE_UNIT_TERM": "Inhalation - unit of product usage",
-        "INDICATION_CODE": "1037351000000105",
-        "LOCATION_CODE": "RJC02",
-        "LOCATION_CODE_TYPE_URI": "https://fhir.nhs.uk/Id/ods-organization-code",
-    }
-
-
-def create_permissions_json(value):
-    return {
-        "all_permissions": {
-            "DPSFULL": ["RSV_FULL", "COVID19_FULL", "FLU_FULL", "MMR_FULL"],
-            "DPSREDUCED": ["COVID19_FULL", "FLU_FULL", "MMR_FULL"],
-            "EMIS": [value, "RSV_FULL"],
-            "PINNACLE": ["COVID19_UPDATE", "RSV_FULL"],
-            "SONAR": "",
-            "TPP": [""],
-            "AGEM-NIVS": [""],
-            "NIMS": [""],
-            "EVA": [""],
-            "RAVS": [""],
-            "MEDICAL_DIRECTOR": [""],
-            "WELSH_DA_1": [""],
-            "WELSH_DA_2": [""],
-            "NORTHERN_IRELAND_DA": [""],
-            "SCOTLAND_DA": [""],
-            "COVID19_VACCINE_RESOLUTION_SERVICEDESK": [""],
-        },
-        "definitions:": {
-            "FULL": "Full permissions to create, update and delete a batch record",
-            "CREATE": "Permission to create a batch record",
-            "UPDATE": "Permission to update a batch record",
-            "DELETE": "Permission to delete a batch record",
-        },
-    }
+class EventName():
+    CREATE = "INSERT"
+    UPDATE = "MODIFY"
+    DELETE_LOGICAL = "MODIFY"
+    DELETE_PHYSICAL = "REMOVE"
+
+
+class Operation():
+    CREATE = "CREATE"
+    UPDATE = "UPDATE"
+    DELETE_LOGICAL = "DELETE"
+    DELETE_PHYSICAL = "REMOVE"
+
+
+class ActionFlag():
+    CREATE = "NEW"
+    UPDATE = "UPDATE"
+    DELETE_LOGICAL = "DELETE"
+    NONE = "NONE"
+
+
+class InfResult:
+    SUCCESS = "Success"
+    PARTIAL_SUCCESS = "Partial Success"
+    FATAL_ERROR = "Fatal Error"
+
+
+class BusRowResult():
+    SUCCESS = "OK"
+    FATAL_ERROR = "Fatal Error"
+    IMMS_NOT_FOUND = "Immunization resource does not exist"
+    NONE = "NONE"
+
+
+class OperationOutcome:
+    IMMS_NOT_FOUND = "Immunization resource does not exist"
+    TEST = "TEST"
+
+
+class OpMsgs:
+    VALIDATION_ERROR = "Validation errors"
+    MISSING_MANDATORY_FIELD = "is a mandatory field"
+    DOSE_QUANTITY_NOT_NUMBER = "doseQuantity.value must be a number"
+    IMM_NOT_EXIST = "Immunization resource does not exist"
+    IDENTIFIER_PROVIDED = "The provided identifier:"
+    INVALID_DATE_FORMAT = "is not in the correct format"
+
+
+class DestinationType:
+    INF = ACK_PREFIX
+    BUS = FORWARDEDFILE_PREFIX
+
+
+class ActionSequence():
```

Comment:
```suggestion
class ActionSequence:
```

This and a few other classes do not need the redundant parentheses.